src/routes/retro-certs.js

```diff
@@ -32,6 +32,7 @@ function createRouter() {
       responseJson.status = AUTH_STRINGS.statusCode.ok;
       responseJson.authToken = formRecord.authToken;
       responseJson.weeksToCertify = Array.from(userRecord.weeksToCertify);
+      responseJson.seekWorkPlan = Array.from(userRecord.seekWorkPlan);
```

Comment:
After some further thought, it's probably fine to just assign userRecord.seekWorkPlan to responseJson.seekWorkPlan (without the Array.from copy). We're not changing any of the values, so it should be safe.